deployment/ansible/roles/driver.letsencrypt/tasks/main.yml

```diff
@@ -1,19 +1,13 @@
 ---
 # Set up certificates on the machine using CertBot
-- name: Install CertBot dependencies
-  apt:
-    pkg: software-properties-common
-    state: present
-
-- name: Add CertBot repository
-  apt_repository:
-    repo: "ppa:certbot/certbot"
-    state: present
 
-- name: Install CertBot
-  apt:
-    pkg: certbot
-    state: latest
+# Need to download certbot directly, since PPA no longer works
+# for this version of Ubuntu
+- name: Download certbot
+  get_url: >
+    url=https://dl.eff.org/certbot-auto
+    dest=/usr/local/bin/certbot
+    mode=0755
```

Comment:
Good call on the `get_url` syntax: I copied it over from another project that was using the older syntax. Updated. 

And I agree that the auto-self-updating isn't the most ideal, and I'd also highly prefer switching back to the PPA when we're able to. Since it takes a couple hours to test this out, in the interest of saving time, would you be able to test out adding some of those flags when you're running through the instance setup, and push a commit if it works out?

Reply:
Will do!